packages/error-reporting/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@google-cloud/error-reporting",
+  "description": "Stackdriver Error Reporting Client Library for Node.js",
+  "main": "index.js",
+  "repository": "GoogleCloudPlatform/google-cloud-node",
+  "scripts": {
+    "test": "nyc --exclude=\"fuzzer.js\" $(npm bin)/mocha ./test/unit/*.js",
```

Comment:
We use istanbul for coverage, and it runs from the root package.json-- see the `cover` and `cover-html` task. To pick it up, you'll have to move the unit tests out from the `test/unit` directory, and up to simply `test`.

Our coverage task looks like this:

```json
{
  "cover": "istanbul cover node_modules/mocha/bin/_mocha --report lcovonly -x 'packages/*/src/*{/*,/**/*}.js' -- --no-timeouts --bail packages/*/test/*.js -R spec"
}
```

The error reporting API is the first that has deviated from a flat `src` directory. If possible, it would be great to see it flattened, but I can understand if that's undesirable. If it's not possible, maybe we can use a convention like leading each directory with a trailing `_`, so it's easier to exclude in the above glob.